src/app.js

```diff
@@ -1,21 +1,38 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
+const http = require('http');
+const url = require('url');
+const fs = require('fs');
 
-module.exports = sum;
+const server = http.createServer((req, res) => {
+  const BASE_URL = `http://${req.headers.host}`;
+  const normalizedUrl = new url.URL(req.url, BASE_URL).pathname;
+  let fileName = normalizedUrl.slice(1).replace('file/', '');
+
+  if (normalizedUrl === '/favicon.ico') {
+    return;
+  }
+
+  if (fileName === '' || fileName === 'file') {
```

Comment:
This condition seems to be wrong. If fileName === 'file' it might mean user requested the 'file' from 'public' folder, not the 'index.html'.